src/_pytest/terminal.py

```diff
@@ -440,6 +440,8 @@ def pytest_runtest_logreport(self, report: TestReport) -> None:
         else:
             markup = None
         self._add_stats(category, [rep])
+        if rep.when == "call" or (rep.when == "setup" and rep.failed):
```

Comment:
Shouldn't this come after `if not letter and not word:`? Because "probably passed setup/teardown" should not be reported again.